.github/merge_rules.json

```
@@ -41,6 +41,27 @@
       "approved_by": ["linbinyu", "kit1980", "IvanKobzarev", "dreiss"],
       "mandatory_checks_name": ["Facebook CLA Check", "Lint"]
    },
+   {
+      "name": "Sparse",
+      "patterns": [
+         "benchmarks/sparse",
+         "c10/util/sparse_bitset.h",
+         "docs/source/sparse.rst",
+         "torch/**/sparse/**",
+         "torch/**/*sparse*",
+         "torch/include/**/*Sparse*",
+         "torch/include/**/*sparse*",
+         "torch/optim/sparse*",
+         "torch/ao/nn/sparse/**",
+         "torch/utils/benchmark/**/*sparse*",
+         "aten/src/ATen/native/ao_sparse/**",
+         "aten/src/ATen/native/sparse/**",
+         "aten/src/ATen/**/*Sparse*",
```

I suggest adding the following patterns if ok with @cpuhrsch :
```suggestion
         "aten/src/ATen/**/*Sparse*",
         "aten/src/ATen/*Sparse*",
         "torch/_masked/**",
         "test/*_masked*",
```

---

Ok with me